install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go

```diff
@@ -73,7 +73,8 @@ type ComponentsSpec struct {
 }
 
 type OauthConfiguration struct {
-	Tag string `json:"tag,omitempty"`
+	DisableSarCheck *bool  `json:"disable-sar-check,omitempty"`
```

Comment:
Can we use camel here?

Reply:
You mean `DisableSarCheck`, rather than `disable-sar-check`?

Reply:
I think we should use the same property as in 1.7.x

Reply:
Ok I meant dromedary case: `disableSarCheck`. All properties for CR use that case

Reply:
Now I see in master it is also called `disable-sar-check` and I can't find that property in 1.7.x :thinking:

Reply:
Consistancy with master I think should be kept.

Reply:
I will change master ...

Reply:
Done.